src/applications/vaos/utils/eligibility.js

```diff
@@ -95,6 +95,17 @@ export async function getEligibilityData(
 
   const results = await promiseAllFromObject(eligibilityChecks);
 
+  if (results.clinics?.length) {
+    // This sorting will be handled by the HealthcareService service once implemented
+    results.clinics = results.clinics.sort(
+      (a, b) =>
+        a.clinicFriendlyLocationName.toUpperCase() <
```

Comment:
I think clinicFriendlyName can be empty for certain clinics, so you'll also need to fallback to clinicName in the sort

Reply:
Fixed to add fallback for clinicName